darren-stoll/mood-tracker
# Chris/implement classnames helper

# What it is
This PR does a little bit of cleanup and adds and implements a `classNames` helper function for simplifying CSS class names.

# How to test it
1. Go to mood entry.
2. Hover over a mood. Notice that the CSS hover state is triggered.
3. Click on a mood. Notice that the selected class is added.
